packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1088,17 +1088,7 @@ test.serial('writeGranulesFromMessage() on re-write saves granule records to Dyn
     apiFormattedPostgresGranule,
     expectedGranule
   );
-  // ES fails here because of bad write logic in <TICKET>
```

Comment:
Confirmed this was part of https://github.com/nasa/cumulus/pull/3242/files#diff-e4b95ee0e3b92433a8e670895c1e8da32d083c3ca64e31761d4bb353cbb791a7L1123 --- commenting here for posterity/future us.